kong/plugins/statsd/log.lua

```diff
@@ -129,11 +128,10 @@ local get_service_id = {
 
 local get_workspace_id = {
   workspace_id   = function()
-    return ws.get_workspace_id()
+    return kong.default_workspace
   end,
   workspace_name = function()
-    local workspace = ws.get_workspace()
-    return workspace.name
+    return "default"
```

Comment:
I expect the hardcode of `workspace` won't work in Enterprise because it supports multiple workspaces.

I see a couple of possible approaches.
a. Moves `workspaces.select_workspace_by_name_with_cache` from EE to CE

b. Continue this but figure out how to make it work in Enterprise.

Reply:
My bad, I was unaware that I'm looking at CE code.

Reply:
As a solution, it would be best if CE provided implementations of `ws.get_workspace_id()` and `ws.get_workspace()` that returned the defaults and not need to access the database instead of having to have two different versions of the `statsd` plugin.

Reply:
I add the function `get_workspace_name()` in `workspaces`. This function will be implemented differently in CE and EE.

In CE, `workspaces.get_workspace()` is only called in test code. I think its existence is only to maintain the same structure as EE. In EE, optimizing the `workspaces.get_workspace()` function will be easier.